docs/source-1.0/spec/core/idl.rst

```diff
@@ -81,135 +82,158 @@ line endings (``\n``). The Smithy ABNF is whitespace sensitive.
 Smithy IDL ABNF
 ---------------
 
-The Smithy IDL is defined by the following ABNF:
+The Smithy IDL is defined by the following ABNF which uses case-sensitive
+string support defined in `RFC 5234 <https://www.rfc-editor.org/rfc/rfc7405>`_.
```

Comment:
Open question: How precisely should the updated IDL 1.0 ABNF match the original IDL 1.0 ABNF?

The current approach copies and modifies the IDL 2.0 grammar, but due to the updated whitespace productions, the updated IDL 1.0 grammar is not an exact match.

If the updated IDL 1.0 needs to be an exact match, then it is probably better to just translate the original ABNF to camelcase and not port the whitespace productions.

Reply:
The grammar was ambiguous before. It should match what IDL 2 uses sans 2.0 specific features, but adding in required commas.